### PR TITLE
Migrate tests to pytest #198

### PR DESCRIPTION
**Pull request by michaelloewenstein**

### CODE REVIEW

File: tests/test_views.py [hunk 1]
```diff
@@ -18,34 +16,33 @@ def _format_response(response: Text = "") -> Union[List, Dict]:
     return json.loads(response)
 
 
-class ProjectTestCase(unittest.TestCase):
+class TestProject:
     BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
-    def setUp(self):
+    def setup_method(self):
```
Comment (avara1986) [inline tests/test_views.py:22]: It's ok, but could be better remove the def setUp and move the initialization of every test to a [conftest](https://docs.pytest.org/en/2.7.3/plugins.html?highlight=re), in example:

## conftest.py
```python
def create_app():
    """Initialize the Flask app, register blueprints and intialize all libraries like Swagger, database, the trace system...
    return the app and the database objects.
    :return:
    """
    ms = MyMicroservice(path=__file__)
    return ms.create_app()

def conf_environment():
    if not os.environ.get("CONFIGMAP_FILE", False):
        os.environ["CONFIGMAP_FILE"] = os.getcwd() + "/config_tests.yml"


@pytest.fixture(scope="session")
def app():
    conf_environment()
    app = create_app()
    return app
```

File: requirements-dev.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Flask-SQLAlchemy==2.4.4
```
Comment (avara1986) [inline requirements-dev.txt:1]: Remember the PR #200 :)

**Review by avara1986 — CHANGES_REQUESTED** [review]

**Comment by premnarayan727** [comment]
#this is app made by me

File: tests/test_views.py (hunk 1 above)
Comment (michaelloewenstein) [reply tests/test_views.py]: OK thanks, giving it another try

**Pull request merged**